plugins/stack-overflow/src/home/StackOverflowQuestions/Content.tsx

```diff
@@ -69,7 +69,7 @@ export const Content = (props: StackOverflowQuestionsContentProps) => {
     <List>
       {value.map(question => (
         <ListItem key={question.link}>
-          <Link to={question.link}>
+          <Link to={question.link} className="link">
```

Comment:
Instead of using `className="link"` and appointing to a `css` file, try to use the variable created on the previous comment.